### PR TITLE
Limit concurrent tasks in Travis to 2

### PR DESCRIPTION
**Pull request by mossprescott**
SBT normally limits tasks overall to the reported number of processors/cores. Come to find out, that number is 32(!) when running in Travis.

Since we believe Travis only uses 1-2 CPUs worth of processor time, resetting the parallel task limit to 2 should preserve any actual parallelism while reducing memory footprint and thrashing.

Running on another branch, this change made the `doc` task complete reliably. It's less clear what it does to the build time.

### CODE REVIEW

File: connector/src/main/scala/quasar/fs/QueryFile.scala [hunk 1]
```diff
@@ -174,7 +174,7 @@ object QueryFile {
     val transforms = Transforms[F]
     import transforms._
 
-    /** Returns the path to the result of executing the given [[LogicalPlan]],
+    /** Returns the path to the result of executing the given `LogicalPlan`,
```
Comment (sellout) [inline connector/src/main/scala/quasar/fs/QueryFile.scala]: Hrmm, why these changes?

1. the original should just be changed to `[[quasar.LogicalPlan]]` and
2. I turned off `-Xfatal-warnings` in scaladoc specifically so we didn’t have to write less-useful scaladoc just because the scaladoc processor was unhappy.

**Comment by mossprescott** [comment]
I tried just adding the package and there was still a (non-fatal) warning, so this change just cleans up the log slightly. In any case the same types appear in the signatures so when you actually look at the generated docs we're not really losing anything.

**Comment by mossprescott** [comment]
The build is looking good and .1 finished in less than an hour which is consistent with what I was seeing on my fork, and would be a massive improvement over recent builds.

**Comment by alissapajer** [comment]
@mossprescott awesome!!
